### PR TITLE
Update DSPAssembler.cpp

### PR DESCRIPTION
**Pull request by Karaulov**
Performance ? :) :) :)

### CODE REVIEW

File: Source/Core/Core/DSP/DSPAssembler.cpp [hunk 1]
```diff
@@ -384,7 +384,8 @@ u32 DSPAssembler::GetParams(char* parstr, param_t* par)
     if (tmpstr == nullptr)
       break;
     tmpstr = skip_spaces(tmpstr);
-    if (strlen(tmpstr) == 0)
+    //if (strlen(tmpstr) == 0)
+    if (!*tmpstr)
```
Comment (degasus) [inline Source/Core/Core/DSP/DSPAssembler.cpp]: I'd use `if (tmpstr[0] == 0)`, and please also drop the comment ;)

Comment (Karaulov) [inline Source/Core/Core/DSP/DSPAssembler.cpp]: ??? more readable? :D :)))

**Comment by lioncash** [comment]
These are effectively equivalent: see [godbolt](https://godbolt.org/g/Ni1Mjv)

**Comment by Karaulov** [comment]
I don't know gcc compiler. 
But in Visual Studio need replace strlen(tmpstr) == 0 to tmpstr[0] == '\0':
http://i.imgur.com/iuAhN98.png

**Pull request closed without merging**